lib/services/elasticsearch.js

```diff
@@ -1318,10 +1362,10 @@ class ElasticSearch extends Service {
         // response.result.get._source
         // => we replace response.result._source with it so that the notifier
         // module can seamlessly process all kind of m* response*
-        for (let i = 0; i < response.items.length; i++) {
+        while ((item = response.items.shift()) !== undefined) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "item" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2703)